project/plugins.sbt

```diff
@@ -3,10 +3,10 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"             % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"             % "3.22.0")
 addSbtPlugin("uk.gov.hmrc"         %% "sbt-distributables"        % "2.5.0")
 addSbtPlugin("uk.gov.hmrc"         % "sbt-accessibility-linter"   % "0.39.0")
-addSbtPlugin("org.playframework"   %% "sbt-plugin"                % "3.0.2")
+addSbtPlugin("org.playframework"   %% "sbt-plugin"                % "3.0.3")
 addSbtPlugin("org.scoverage"       %% "sbt-scoverage"             % "2.0.11")
```

Comment:
I had the same compatibility issue, found that upgrading this to 2.0.12 fixed it for me
`addSbtPlugin("org.scoverage"          % "sbt-scoverage"             % "2.0.12")`

Reply:
Thanks, that works for me too, so will update.